lab_2/main.py

```diff
@@ -30,30 +30,107 @@ def read_ecg_rr_file(file_path: Path):
 
 # Lab 2 implementation goes below
 
+# def signal_check(signal):
+#     if type(signal) != list or len(signal) == 0:
+#         return None
+#     for s in signal:
+#         if type(s) != float:
+#             return None
+
```

Comment:
### Merging changes via merge commit

`git fetch --all`

`git merge origin/main`

Conflicts to be resolved

### Merging changes via rebase (default)

`git pull --rebase`

`git rebase --abort` - to stop rebasing